packages/patternlab/styleguide/source/_patterns/03-organisms/by-author/social-links-bar.twig

```diff
@@ -1,4 +1,5 @@
 <section class="ma__social-links-bar">
+  <h2 class="ma__visually-hidden">Social media links</h2>
```

Comment:
When a comp heading is available, this heading level should be `h3`.
The sample page in the ticket doesn't have comp heading for the section, thus this heading level is `h2` as shown.
Please make the heading level adjustable based on the availability of the comp heading.

Reply:
I've identified the level and passed to the template, can you help me adjust the templates to reflect the level correctly?

Reply:
@arthurbaghdas Thank you for your work.  

I think **packages/patternlab/styleguide/source/_patterns/01-atoms/04-headings/comp-heading.twig** is a good reference for setting a heading level.

First, in Drupal, the heading level that you set up as `$variables['level'] = $level;` in the .theme file is passed to Mayflower as a part of **socialLinksBar**:  `socialLinksBar.level`

At the top of the template, set up a variable for the heading level.  Set a default value just in case nothing's passed as a level value.  Then, the variable can be placed in the heading element.
```
{# In case no level value is available, its default value is set as 3. #}
{% set headingLevel = socialLinksBar.level ? socialLinksBar.level : "3" %}
<div class="ma__social-links-bar">
...
   <h{{ headingLevel }} class="ma__visually-hidden">Social media links</h{{ headingLevel }}
...
</div>
```
Let me know for any questions.  Thanks!